cms/templates/js/course-outline-modal.underscore

```diff
@@ -1,14 +1,6 @@
-<%
-var enable_proctored_exams = enable_proctored_exams;
-var enable_timed_exams = enable_timed_exams;
-%>
-
 <div class="xblock-editor" data-locator="<%= xblockInfo.get('id') %>" data-course-key="<%= xblockInfo.get('courseKey') %>">
-    <% if (!( enable_proctored_exams || enable_timed_exams )) { %>
-        <div class="message modal-introduction">
-            <p><%- introductionMessage %></p>
-        </div>
-    <% } %>
+    <div class="message modal-introduction">
+        <p><%- introductionMessage %></p>
```

Comment:
We were asked to remove the Introduction Message which starts with "Change the settings for ..." (https://github.com/edx/edx-platform/pull/10569#issuecomment-163691095) when showing multiple tabs. Removing the above checks always shows the introduction message. I think we should probably still show the introduction message when tab-count==1, and should not show it when >= 2 tabs